### ISSUE TITLE
Issue to trigger new releases

### ISSUE THREAD

**Issue by ablaom** [JuliaAI/MLJMultivariateStatsInterface.jl#4]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@DilumAluthge I'm still confused how to get registration via issue to work. I already did a first regsitration using JuliaHub. What extra steps are necessary?

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by OkonSamuel** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@OkonSamuel Oops. As this is breaking this should have been a minor version. I will make a PR to yank it from the registry.

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

**Comment by ablaom** [comment]
@JuliaRegistrator register

release notes:

- extend [compat] for MLJModelInterface to include 1.0

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Bump Julia requirement to 1.6
- (**mildly breaking**) Change the default value of `inverse` in `kernelPCA` to `true` (#28)
- Internally use adjoints instead of transposes on matrixified table input, except in LDA models where `MultivariateStats.jl` lacks adjoint support (partially addresses #27)
- Add support for MultivariateStats 0.9 (#29) @testercwt

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- (**breaking**) Rename the `class_means` key in reports  to `projected_class_means`, for consistency with key name in `fitted_params(model, ...)`.  Affects `LDA` and its relatives (#47).
- (**breaking**) Rename `out_dim` in hyper-parameters and reports with `outdim` for consistency with use in other models. Effects `ICA`, and `LDA` and relatives of `LDA` (#47).
- (**breaking**) Rename the `pratio` hyperparameter in `PCA` to `variance_ratio` as more informative (#47)
- (**breaking**) Rename `nc` in reports to `nclasses` as more informative. Affects  `LDA` and its relatives (#47)
- (**breaking**) Rename hyper-parameter `k` in `ICA` to `outdim` for consistency with other models and report (#47)
- (**docs**) Add detailed MLJ-compliant document strings to all models (#39) @josephsdavid, @ablaom

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

Bumps the MultivariateStats requirement to 0.10, which forces some breaking changes (#51; @OkonSamuel):

- (**breaking**) The output of `fitted_params` and `report` for LDA models has changed. See the doc-strings for details. 
- (**breaking**) Interpreting some output (now and previously) needs an ordering of target classes, and this ordering may no longer agree with `levels(y)`. However the class label order adopted is now explicitly output by `fitted_params`.  
- (**breaking**) In `BayesianLDA` and `BayesianSubspaceLDA` target class priors may no longer be specified as vectors. They must either be dictionaries or `UnivariateFinite` objects from CategoricalDistributions.jl.

**Comment by OkonSamuel** [comment]
@JuliaRegistrator register branch=master

release notes:

- add `loadings` to report for `PCA` model.

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Bump the compatibility requirements, CategoricalDistributions="1"

**Comment by ablaom** [comment]
@JuliaRegistrator register branch=master

Release notes:

- Bump the compatibility requirements, CategoricalDistributions="1"
